blog/2024-08-27-wasmcloud-1.2-improved-streams-automatic-config-updates/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: "wasmCloud 1.2: Improved streams, automatic config updates for providers, and more"
+authors: [brooksmtownsend]
+image: './images/wasmcloud-1.2.png'
+slug: 2024-08-27-wasmcloud-1.2-improved-streams-automatic-config-updates
+date: "2024-08-27"
+description: "Check out the enhancements in wasmCloud 1.2"
+---
+
+![wasmcloud 1.2](./images/wasmcloud-1.2.png)
+
+wasmCloud 1.2 officially closes out the remaining issues from the [Q2 Roadmap](https://github.com/orgs/wasmCloud/projects/7/views/10)! This release is also the first of our quicker release cadence, so expect to see some great changes&mdash;but not quite as many as wasmCloud 1.1.
```

Comment:
This release is the first of our quicker release cadence - feels like this should be cadences (given we're suggesting a series). But that reads weirdly.

Suggest: This is the first release since we decided to quicken our release cadence. / This is the first release since we adopted our new, accelerated release cadence.

Reply:
I like "new, accelerated release cadence" -- updated!